controllers/os-ubuntu-alicloud/pkg/generator/templates/cloud-init.ubuntu-ali.template

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+{{- define "put-content" -}}
+cat << EOF | base64 -d > '{{ .Path }}'
+{{ .Content }}
+EOF
+{{- end -}}
+
+{{- if .Bootstrap }}
+#Disable upgrade related services
+systemctl disable locksmithd
+systemctl stop locksmithd
+systemctl disable update-engine
+systemctl stop update-engine
+
+#install needed packages
+apt update
+apt install -y docker.io socat nfs-common logrotate jq
```

Comment:
In the non-alicloud ubuntu controller a package called `policykit-1` is installed, too.
Is it necessary on alicloud?